src/tests.c

```diff
@@ -44,6 +44,21 @@ static int all_bytes_equal(const void* s, unsigned char value, size_t n) {
     return 1;
 }
 
+/* Debug helper for printing arrays of unsigned char. */
+static void print_buf(const unsigned char *buf, size_t len) {
+    size_t i;
+    printf("{");
```

Comment:
This would be extra useful if this helper would also output the name of the printed variable. If `print_buf` was a  macro we could do it like this:
```suggestion
    printf("%s = {", #buf);
```